command/src/logging.rs

```diff
@@ -682,6 +682,6 @@ impl std::fmt::Display for Rfc3339Time {
 }
 
 pub fn now() -> (Rfc3339Time, i128) {
-  let t = time::PrimitiveDateTime::now();
+  let t = std::time::SystemTime::now().into();
```

Comment:
why change the type here if we're comparing with a `PrimitiveDateTime` right after?

Reply:
time::PrimitiveDateTime is deprecated.